extensions/ql-vscode/src/telemetry.ts

```diff
@@ -28,6 +28,38 @@ export enum CommandCompletion {
   Cancelled = "Cancelled",
 }
 
+export type ErrorType =
```

Comment:
Oh...I see you already have this error type. Does it make sense to use this on the options object above?

Reply:
I think this comment is referring to `notificationMessage` and there's maybe been a misunderstanding on one of our parts. So I'll leave this bit and let us resolve the other comment first.